TypeInjections/Tool/Tool.fsproj

```diff
@@ -15,24 +15,13 @@
 
 
     <ItemGroup>
-        <Reference Include="Boogie.Core, Version=2.15.x, Culture=neutral, PublicKeyToken=null">
-            <HintPath>..\..\dafny\Boogie.Core.dll</HintPath>
-        </Reference>
-        <Reference Include="Dafny, Version=3.6.x, Culture=neutral, PublicKeyToken=null">
-            <HintPath>..\..\dafny\Dafny.dll</HintPath>
-        </Reference>
-        <Reference Include="DafnyPipeline, Version=3.6.x, Culture=neutral, PublicKeyToken=null">
-            <HintPath>..\..\dafny\DafnyPipeline.dll</HintPath>
-        </Reference>
-        <Reference Include="Boogie.BaseTypes, Version=2.15.x, Culture=neutral, PublicKeyToken=null">
-            <HintPath>..\..\dafny\Boogie.BaseTypes.dll</HintPath>
-        </Reference>
-        <Reference Include="Boogie.ExecutionEngine, Version=2.15.x, Culture=neutral, PublicKeyToken=null">
-            <HintPath>..\..\dafny\Boogie.ExecutionEngine.dll</HintPath>
-        </Reference>
-        <Reference Include="Boogie.CodeContractsExtender, Version=2.15.x, Culture=neutral, PublicKeyToken=null">
-            <HintPath>..\..\dafny\Boogie.CodeContractsExtender.dll</HintPath>
-        </Reference>
+        <PackageReference Include="Boogie.CodeContractsExtender" Version="2.15.7" />
+        <PackageReference Include="Boogie.Core" Version="2.15.7" />
+        <PackageReference Include="Boogie.ExecutionEngine" Version="2.15.7" />
+        <PackageReference Include="CommandLineParser.FSharp" Version="2.9.2-ci-210" />
+        <PackageReference Include="DafnyPipeline" Version="3.6.0.40511" />
+        <PackageReference Include="LibGit2Sharp" Version="0.27.0-preview-0182" />
+        <PackageReference Include="Microsoft.Z3" Version="4.10.1" />
```

Comment:
Dafny uses version 4.8.5, but I don't know if we even need this here? We don't call the verifier. Even if we did, Dafny requires z3 to be installed (well it installs it via makefile).